tests/_client.lua

```diff
@@ -24,16 +24,21 @@ local function open_window(class, title, options)
         default_height = options.default_height or 100,
         title          = title
     }
+    if options.resize_increment or options.custom_titlebar then
+        -- These require Gtk3, but may fail with an obscure message with Gtk2.
+        -- Produce a better error message instead.
+        assert(tonumber(require("lgi").Gtk._version) >= 3, "Gtk 3 required, but not found")
```

Comment:
there is a proper fix for gtk 3.0 / 4.0 conflict, however i don't have time right now to finish it, but i posted 2 patches inside the issue: https://github.com/awesomeWM/awesome/issues/3849

Reply:
no worries regarding gtk 2 - there are other tests which already depend on gtk3